### ISSUE TITLE
Proximity Integration "tolerance" units unclear

### ISSUE THREAD

**Issue by matthewcbyington** [home-assistant/home-assistant.github.io#25641]
### Feedback

I was trying to setup the Proximity integration. I'm unclear, based on the documentation, of whether the "unit_of_measurement" applies to the tolerance?

The tolerance says "meters" in its description.

As a result I setup the integration as follows:

## Proximity ##
proximity:
  home:
    ignored_zones:
      - work
    devices:
      - person.matt
      - person.serena
    tolerance: 100
    unit_of_measurement: mi

However when leaving the house and going a few miles away, nothing updated and it still said "arrived" as the state.

My gut tells me this is because I've set the tolerance to 100 miles, but the documentation could be more clear on what the unit of measurement actually applies to - and I think the description of "tolerance" should be updated as well.

Thanks so much!

Matt

### URL

https://www.home-assistant.io/integrations/proximity/

**Comment by frenck** [comment]
The proximity sensor measures the distances to a zone, as per documentation:

![CleanShot 2023-01-10 at 12 37 48](https://user-images.githubusercontent.com/195327/211542269-53c8bc05-7e68-45e6-84a2-ab6520e7a0b5.png)

The unit of measurement configuration defines the unit in which this distance is represented in the sensor state:

![CleanShot 2023-01-10 at 12 38 35](https://user-images.githubusercontent.com/195327/211542404-ab79f7bc-fe2b-4562-a957-754b09c272b9.png)

The unit of measurement is not related to tolerance, as the documentation states that it needs to be in meters, as documented:

![CleanShot 2023-01-10 at 12 39 27](https://user-images.githubusercontent.com/195327/211542484-c54ed6c5-f509-42ea-b7c1-dbca54d91e9e.png)

../Frenck

**Comment by matthewcbyington** [comment]
@frenck thank you for taking the time to comment I appreciate it! It’s weird, when I set tolerance to 100, the entity did not update even when I traveled 4 miles. When I set the tolerance to 2, the entity updated every 2 miles. That’s why I thought there might be an issue. For me the tolerance didn’t seem to be in meters when I set unit of measurement to Miles. Not sure if it was just me, but just commenting as an fyi.
